src/features/metadata-based-view/constants.js

```diff
@@ -0,0 +1,15 @@
+// @flow strict
+
+export const CANCEL_ICON_TYPE = 'cancel';
+export const EDIT_ICON_TYPE = 'edit';
+export const SAVE_ICON_TYPE = 'save';
```

Comment:
Should these define their literal flow type? i.e. `export const CANCEL_ICON_TYPE: 'cancel' = 'cancel'`